content/blog/run-your-own-rss-server/index.md

```diff
@@ -34,7 +34,7 @@ As I mentioned, the service is packaged as a Docker container that runs on port
 * `ADMIN_USERNAME` and `ADMIN_PASSWORD`, which we'll use to sign into the service for the first time, and
 * flags for `CREATE_ADMIN` and `RUN_MIGRATIONS`, which denote whether to create the administrative user and run database migrations on startup.
 
-We'll set these properties in the course of developing our program. The service itself, as a container, can easily be run on [AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html), so we can use the [`@pulumi/awsx`]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}) package to declare it, and for the database, we'll use [`@pulumi/aws`]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws" >}}) to provision a small [RDS instance of PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html). And finally, to make the Miniflux service publicly accessible (initially over HTTP; we'll switch to HTTPS later), we'll use an AWS [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html), since we'll only need to be able to route traffic by ports and protocols.
+We'll set these properties in the course of developing our program. The service itself, as a container, can easily be run on [AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html), so we can use the [`@pulumi/awsx`]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}) package to declare it, and for the database, we'll use [`@pulumi/aws`]({{< relref "//docs/reference/pkg/aws" >}}) to provision a small [RDS instance of PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html). And finally, to make the Miniflux service publicly accessible (initially over HTTP; we'll switch to HTTPS later), we'll use an AWS [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html), since we'll only need to be able to route traffic by ports and protocols.
```

Comment:
There seems to have been a find/replace error here. There's an extra leading slash.

```suggestion
We'll set these properties in the course of developing our program. The service itself, as a container, can easily be run on [AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/AWS_Fargate.html), so we can use the [`@pulumi/awsx`]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}) package to declare it, and for the database, we'll use [`@pulumi/aws`]({{< relref "/docs/reference/pkg/aws" >}}) to provision a small [RDS instance of PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html). And finally, to make the Miniflux service publicly accessible (initially over HTTP; we'll switch to HTTPS later), we'll use an AWS [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/network-load-balancers.html), since we'll only need to be able to route traffic by ports and protocols.
```

...but I think we should revert this change because we are referring to the `@pulumi/aws` npm package here, so maybe we should leave the link as it was before, to the nodejs API docs for `aws`?